cloudformation/coast-cfn.yaml

```diff
@@ -784,6 +784,8 @@ Resources:
           import urllib3
           import json
           import logging
+          from urllib.request import Request, urlopen
+          from urllib.parse import urlencode
```

Comment:
What if instead of importing urllib, we used the grafana_api function instead? it will perform a similar work and relies only on urllib3.